release-tool

```diff
@@ -525,13 +532,16 @@ build() {
     done
 
     init
-    checkWorkingTreeClean
 
-    OUTPUT_DIR="$(realpath "$OUTPUT_DIR")"
-    
-    logInfo "Checking out release tag '${TAG_NAME}'..."
-    git checkout "$TAG_NAME"
+    if ! $IGNORE_REPO; then
```

Comment:
I don't like this. The repo needs to be clean in order to perform a git checkout. If the existing check fails on Windows, fix the check, don't disable it.

Reply:
Sorry for the confusion. The ignore repo flag has nothing to do with windows. It just lets you use the release tool to build and deploy (sign) without a clean repo. It's convenance for quick tests of the deploy pipeline.

Reply:
This still looks a little dirty to me. The --version parameter is required, because the release tool always checks out a specific tag. If you want a shortcut, this should be done by specifying something like --version none or --version current which tells the release-tool not to check out any tags and just use the current (dirty) working tree.